src/camera.rs

```diff
@@ -80,14 +17,14 @@ pub fn camera_follow_player(
         .max_by(|ax, bx| ax.total_cmp(bx));
 
     if let Some(max_player_x) = max_player_x {
-        let (mut camera, panning) = camera_query.single_mut();
+        let camera = camera_query.single();
 
         let max_player_x_diff =
             max_player_x - camera.translation.x - game_meta.camera_move_right_boundary;
 
         if max_player_x_diff > 0. {
             // The x axis is handled by the parallax plugin.
-            camera.translation.y = consts::GROUND_Y + panning.offset.y;
+            // The y axis value doesn't change.
```

Comment:
These changes should still be made, but most of the other changes were already done in #100.